src/openrct2/platform/shared.c

```diff
@@ -822,4 +808,12 @@ uint8 platform_get_currency_value(const char *currCode) {
 void core_init()
 {
 	bitcount_init();
+	
+#if defined(__APPLE__) && !defined(MACOS_SIERRA_ONWARDS)
```

Comment:
Whoops! Clearly doing that stuff late at night was a bad move. Thanks for fixing!